packages/chat/src/bubble-info.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+import { BubbleInfo } from './bubble-info'
+
+export default {
+  title: 'chat-bubble / BubbleInfo',
+  component: BubbleInfo,
+  argTypes: {
+    date: {
+      control: {
+        type: 'date',
+      },
+    },
+  },
+}
```

Comment:
```suggestion
} as ComponentMeta<typeof BubbleInfo>
```
아마 argTypes에도 타입 추론이 가능할겁니다.

Reply:
https://github.com/titicacadev/triple-frontend/pull/2238
074aed96d5db5b2b15a09ad415aece0a0bd5aa84 리뷰주신부분 이쪽에서 반영했습니다 🙇🏻